test-data/unit/check-unreachable-code.test

```diff
@@ -841,6 +841,38 @@ def baz(x: int) -> int:
         return x
 [builtins fixtures/exception.pyi]
 
+[case testUnreachableFlagExcepts]
+# flags: --warn-unreachable
```

Comment:
We need the same test (or simplified) without this flag to make sure it does not produce any errors in normal mode.

Reply:
Where should this test go? Still in `check-unreachable-code.test`? I cannot find the equivalent for other features in this file. Into `check-statements.test?`

In `check-statements.test` there is already a test covering the use-case.

```
[case testMultipleExceptHandlers]
import typing
try:
    pass
except BaseException as e:
    pass
except Err as f:
    f = BaseException() # Fail
    f = Err()
class Err(BaseException): pass
[builtins fixtures/exception.pyi]
[out]
main:7: error: Incompatible types in assignment (expression has type "BaseException", variable has type "Err")
```

Reply:
Yes, `check-unreachable-code.test` is fine I guess 🙂